src/pages/components/content-switcher/code.mdx

```diff
@@ -4,34 +4,180 @@ description: Content switchers manipulate the content shown following an exclusi
+| selected  | `bool`              | –        | `false`        | Whether your Switch is selected. Reserved for usage in ContentSwitcher                                                  |
+| text      | `string`            | `true`   | 'Provide text' | Provide the contents of your Switch                                                                                     |
+| onClick   | `func`              | –        | –              | A handler that is invoked when a user clicks on the control. Reserved for usage in ContentSwitcher                      |
+| onKeyDown | `func`              | –        | –              | A handler that is invoked on the key down event for the control. Reserved for usage in ContentSwitcher                  |
+| className | `string`            | –        | –              | Specify an optional className to be added to your Switch                                                                |
+| index     | `number`            | –        | –              | The index of your Switch in your ContentSwitcher that is used for event handlers. Reserved for usage in ContentSwitcher |
+| name      | 'string' / 'number' | –        | –              | Provide the name of your Switch that is used for event handlers                                                         |
+
+### Switch selected
+
+The `selected` is used to specify weather the specific `Switch` is selected.
+
+```jsx
+<ContentSwitcher onChange={function noRefCheck() {}} selectionMode="manual">
+  <Switch
+    selected={false}
+    onClick={function noRefCheck() {}}
+    onKeyDown={function noRefCheck() {}}
+    text="First section"
+  />
+  <Switch
+    selected={false}
+    onClick={function noRefCheck() {}}
+    onKeyDown={function noRefCheck() {}}
+    text="Second section"
+  />
+  <Switch
+    selected={false}
+    onClick={function noRefCheck() {}}
+    onKeyDown={function noRefCheck() {}}
+    text="Third section"
+  />
 </ContentSwitcher>
-`}</ComponentVariant>
-</ComponentDemo>
+```
+
+### Switch text
+
+The `text` prop is used to provide the label displayed within a `Switch` component.
+
+```jsx
+<ContentSwitcher onChange={function noRefCheck() {}} selectionMode="manual">
+  <Switch
+    onClick={function noRefCheck() {}}
+    onKeyDown={function noRefCheck() {}}
+    text="First section"
+  />
+  <Switch
+    onClick={function noRefCheck() {}}
+    onKeyDown={function noRefCheck() {}}
+    text="Second section"
+  />
+  <Switch
+    onClick={function noRefCheck() {}}
+    onKeyDown={function noRefCheck() {}}
+    text="Third section"
+  />
+</ContentSwitcher>
+```
+
+## References
+
+Each section component itself is rendered as a `button` and is considered an interactive element. If you render an interactive element inside of this component, then it becomes nested inside of the `button`. Common examples of this are links. Rendering interactive content inside of the `Switch` is not reachable via a keyboard or screen reader.
```

Comment:
Is section here supposed to be `Switch`? 

Also, for the last sentence, since interactive content is not reachable via a keyboard or screen reader, should we add something like

> Rendering interactive content inside of the `Switch` is not reachable via a keyboard or screen reader, so we do not recommend this

Reply:
Yes thats a good addition.